server/src/db/db-config.js

```diff
@@ -0,0 +1,41 @@
+import Sequelize from 'sequelize';
+
+//make path for production environment
+
+let db; 
+
+if (process.env.DATABASE_URL) {
+  db = new Sequelize(process.env.DATABASE_URL, {
+    host: process.env.DATABASE_URL.split(':')[2],
+    protocol: 'postgres',
+    dialect: 'postgres'
+
+    , pool: {
+      max: 5,
+      min: 0,
+      idle: 10000
+    }
+  })
+//and dev environment
```

Comment:
Instead of using an if/else block here, can we define the host using a ternary statement checking the value of process.env.DATABASE_URL?

Eg. host = process.env.DATABASE_URL ? process.env.DATABASE_URL.split(':')[2] : 'localhost';